example/README.md

```diff
@@ -0,0 +1 @@
+# example
```

Comment:
@felixminom let's add some basic instructions about how to deploy this app to a VM using `hapistrano`, please